.github/workflows/test.yml

```diff
@@ -50,6 +50,10 @@ jobs:
           # Check cross-compiling macOS binaries.
           env GOOS=darwin GOARCH=amd64 go build -v ./...
           env GOOS=darwin GOARCH=arm64 go build -v ./...      
+          
+          # Check cross-compiling Linux binaries.
+          env GOOS=linux GOARCH=amd64 go build -v ./...
+          env GOOS=linux GOARCH=arm64 go build -v ./...  
```

Comment:
(I don't think we don't do this in this PR but) do we have a plan to support 386 and arm?

Reply:
I would love to! It should be really easy. Just write `sys_linux_{386+arm}.s` and `go_linux_{386+arm}.go`. There are also examples for those GOARCHs for windows which could be used as inspiration. However, I don't have systems to test these with so I can't do it :/

Reply:
We should be able to use QEMU to confirm, right?

Reply:
True!

Reply:
> There are also examples for those GOARCHs for windows which could be used as inspiration.

I missed the word 'Windows'. Testing Windows for arm seems a little tricky.

Anyway I'm looking forward to adding more architectures in the future 😄